tracsubtickets/web_ui.py

```diff
@@ -172,8 +217,8 @@ def get_children(self, parent_id, depth=0):
         children = {}
 
         for parent, child in self.env.db_query("""
-                SELECT parent, child FROM subtickets WHERE parent=%s
-                """, (parent_id, )):
+                SELECT parent, child FROM subtickets WHERE parent IN %s
+                """, ([parent_id], )):
```

Comment:
After the changes, it doesn't work with SQLite and PostgreSQL. They should work with all databases without the changes.